rules/mocha.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  plugins: [
+    'mocha',
+  ],
+  rules: {
+    'mocha/no-exclusive-tests': 'error',
+    'mocha/no-skipped-tests': 'off',
+    'mocha/no-pending-tests': 'error',
+    'mocha/handle-done-callback': 'error',
+    'mocha/no-synchronous-tests': 'off',
+    'mocha/no-global-tests': 'error',
+    'mocha/no-return-and-callback': 'error',
+    'mocha/valid-test-description': 'off',
+    'mocha/valid-suite-description': 'off',
+    'mocha/no-sibling-hooks': 'off',
```

Comment:
well, Mocha does officially support having multiple hooks of the same kind so I didn't see a reason to enable it. the `setupComponentTest()` helper etc. actually rely on that stuff, although the ESLint rule most likely wouldn't be able to catch that.